service/src/ConstConfigIOGroup.cpp

```diff
@@ -0,0 +1,424 @@
+/*
+ * Copyright (c) 2015 - 2022, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "ConstConfigIOGroup.hpp"
+
+#include <cmath>
+
+#include <iostream>
+#include <iterator>
+#include <sstream>
+
+#include "geopm/Helper.hpp"
+#include "geopm/PlatformTopo.hpp"
+#include "geopm/Exception.hpp"
+#include "geopm/Agg.hpp"
+#include "geopm_topo.h"
+#include "config.h"
+
+using json11::Json;
+
+namespace geopm
+{
+    const std::string ConstConfigIOGroup::M_PLUGIN_NAME = "CONST_CONFIG";
+    const std::string ConstConfigIOGroup::M_SIGNAL_PREFIX = M_PLUGIN_NAME + "::";
+    const std::map<std::string, ConstConfigIOGroup::m_signal_desc_s>
+        ConstConfigIOGroup::M_SIGNAL_FIELDS = {
+            {"description", {Json::STRING, true}},
+            {"units", {Json::STRING, true}},
+            {"domain", {Json::STRING, true}},
+            {"aggregation", {Json::STRING, true}},
+            {"values", {Json::ARRAY, true}}
+        };
+
+    ConstConfigIOGroup::ConstConfigIOGroup()
+    {
+        // TODO
```

Comment:
I think this is sufficient as an empty constructor, since it just means we'll expose no signals, right?

Reply:
See my reply to Brad's comment.